frameworks_test.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/kudobuilder/kudo/pkg/test"
+)
+
+// Launch the KUDO tests.
+// To add more tests, add them to TestDirs belowe.
```

Comment:
Could we just look for all test folders instead of having these hardcoded?

Reply:
Changed to a glob for now. Will merge once tests pass.